tests/network/vmi_networking.go

```diff
@@ -511,6 +511,34 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 
 			Expect(err).ToNot(HaveOccurred())
 		})
+
+		It("[test_id:xxxx]should allow NTP servers' IPv4 addresses which contain leading zeros", func() {
```

Comment:
First, you do not need the `test_id`, please drop it.

Secondly, could you please clarify what is the advantage to have another dedicated e2e test for this edge case? e2e tests in general are costly because the creation and termination of a VMI takes a significant time.

How about just add to the prev test a 3rd address, e.g.: `172.0.0.099` ? You can place it in a const and explain in a comment above it that it is intentional.

Reply:
I don't think we have to keep a dedicated test for this case.
Although it would be nice to have a focused warning when this specific feature fails, I think the cost is too high for the value it could provide.

It was a response to @ormergi 's comment: https://github.com/kubevirt/kubevirt/pull/6986#pullrequestreview-840082503